test/unit/serializer.test.js

```diff
@@ -43,6 +43,30 @@ test('Basic', (t) => {
   t.same(s.deserialize(json), obj);
 });
 
+test('Long numerals', (t) => {
+  t.plan(6);
+  const s = new Serializer();
+  const longPositive = BigInt(Number.MAX_SAFE_INTEGER) * 2n; // eslint-disable-line no-undef
+  const longNegative = BigInt(Number.MIN_SAFE_INTEGER) * 2n; // eslint-disable-line no-undef
```

Comment:
Can we also have a specific test case? Say "const number = 102931203123987", So when can avoid `a == a` test bug that's always true regardless of the logic.